sys/netpfil/ipfilter/netinet/ip_ftp_pxy.c

```diff
@@ -1715,7 +1715,7 @@ ipf_p_ftp_eprt4(ipf_ftp_softc_t *softf, fr_info_t *fin, ip_t *ip, nat_t *nat,
 		return (0);
 	if (c != delim)
 		return (0);
-	addr |= addr;
+	/* addr |= (addr << 0); */
```

Comment:
Commenting out statements can be prone to undesirable results. How about a #if 0 block instead?